conf/zapiperf/cdot/9.8.0/workload.yaml

```diff
@@ -6,6 +6,9 @@ query:                    workload
 object:                   qos
 
 instance_key:             uuid
+workload_class:
```

Comment:
might be worth adding a small comment above `workload_class` - something like
# filter workloads by class. one of autovolume|system_defined|user_defined

Reply:
probably worth a comment